src/ci/docker/x86_64-gnu-tools/checkregression.py

```diff
@@ -44,5 +44,6 @@
             if not (verb == 'regressed' and tool in REGRESSION_OK):
                 regressed = True
 
-    if regressed:
+    # don't abort the build if we're just checking for changes
+    if regressed and verb == 'regressed':
```

Comment:
This means the `verb == 'regressed'` check in line 44 is useless and is equivalent to `if tool not in REGRESSION_OK` there